dxr/query.py

```diff
@@ -102,8 +102,27 @@ def results(self, offset=0, limit=100):
         # will OR the elements of the inner lists and then AND those OR balls
         # together.
```

Comment:
We should move this comment down to merge with your "Some filters" comment; it hasn't been in the right place for quite some time.